spec/features/items/index_spec.rb

```diff
@@ -47,15 +47,13 @@
         expect(page).to have_css("img[src*='#{@pull_toy.image}']")
       end
 
-      within "#item-#{@dog_bone.id}" do
-        expect(page).to have_link(@dog_bone.name)
-        expect(page).to have_content(@dog_bone.description)
-        expect(page).to have_content("Price: $#{@dog_bone.price}")
-        expect(page).to have_content("Inactive")
-        expect(page).to have_content("Inventory: #{@dog_bone.inventory}")
-        expect(page).to have_link(@brian.name)
-        expect(page).to have_css("img[src*='#{@dog_bone.image}']")
-      end
+        expect(page).to have_no_link(@dog_bone.name)
+        expect(page).to have_no_content(@dog_bone.description)
+        expect(page).to have_no_content("Price: $#{@dog_bone.price}")
+        expect(page).to have_no_content("Inactive")
+        expect(page).to have_no_content("Inventory: #{@dog_bone.inventory}")
+        expect(page).to have_no_css("img[src*='#{@dog_bone.image}']")
+
     end
```

Comment:
Should we check that the links of this page are going to the right places?

Reply:
yes, good call and that has been added now, as well